src/Symfony/Component/Mime/Part/BodyFile.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mime\Part;
+
+use Symfony\Component\Mime\MimeTypes;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+final class BodyFile
```

Comment:
I think it'd be better for extensibility of the classes consuming this one to not make it final.

Reply:
I don't see why we would want to extend this file.

Reply:
It's not really about extending this file, but about being able to provide other implementations to the constructor of eg DataPart. Using `final` creates hard-coupling between both classes. An alternative would be to add a BodyFileInterface, but I'm not sure it's worth it.